checkov/dockerfile/checks/graph_checks/RunYumConfigManagerSslVerify.yaml

```diff
@@ -0,0 +1,11 @@
+metadata:
+  id: "CKV2_DOCKER_15"
+  name: "Ensure that the yum and dnf package managers are not configured to disable SSL certificate validation via the 'sslverify' configuration option"
+  category: "APPLICATION_SECURITY"
+definition:
+  cond_type: attribute
+  resource_types:
+    - RUN
+  attribute: value
+  operator: not_regex_match
+  value: ".*(((yum)|(dnf))(\\s+|-)config-manager[^\\|&;]*\\s+--setopt=['\"]?([^=\\s]+\\.)?sslverify['\"]?=((0)|('0')|(\"0\")|([Ff]alse)|('[Ff]alse')|(\"[Ff]alse\")|([Nn]o)|('[Nn]o')|(\"[Nn]o\"))['\"]?).*"
```

Comment:
you could use the global marker `(?i)` to make the regex case insensitive, then there is not need for `[Ff]alse`
```
"(?i).*(((yum)..."
```